File: src/test/java/us/fatehi/palindrome/BetterPalindromeTest.java
package us.fatehi.palindrome;


import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

public class BetterPalindromeTest
{

  private final Palindrome p = new Palindrome();

  @Test
  public void boundaryConditionHappy()
  {
    assertTrue(p.isPalindrome("oo"));
  }

  @Test
  public void boundaryConditionNegative()
  {
    assertFalse(p.isPalindrome("ah"));
  }

}

File: C#/TestPalindrome/BetterPalindromeTest.cs
using NUnit.Framework;
using us.fatehi.palindrome;

namespace us.fatehi.palindrome.test
{

    [TestFixture]
    public class BetterPalindromeTest
    {
        private readonly Palindrome _p = new Palindrome();

        [Test]
        public void BoundaryConditionHappy()
        {
            Assert.That(_p.IsPalindrome("oo"));
        }

        [Test]
        public void BoundaryConditionNegative()
        {
            Assert.That(_p.IsPalindrome("ah"), Is.False);
        }
    }
}